bbl-up/task

```diff
@@ -103,7 +103,7 @@ function main() {
       cp -r ${root_dir}/bbl-config/${BBL_CONFIG_DIR}/. .
     fi
 
-    eval bbl --debug up \
+    eval bbl up --debug \
```

Comment:
Could you please revert this change? The `--debug` flag is a global option according to the bbl help doc.